test/suites/backup.sh

```diff
@@ -250,6 +250,51 @@ test_backup_import() {
     lxc start b2
     lxc delete --force b2
   fi
+
+  default_pool="$(lxc profile device get default root pool)"
+  default_path="$(lxc profile device get default root path)"
+
+  # Create new storage pools
+  lxc storage create pool_1 dir
+  lxc storage create pool_2 dir
+
+  # Export created container
+  lxc init testimage b3 -s pool_1
+  lxc export b3 "${LXD_DIR}/c3.tar.gz"
```

Comment:
Can we either call the container `c3` or call the file `b3`? The mix of the two is confusing :)